homeassistant/components/updater.py

```diff
@@ -0,0 +1,51 @@
+"""
+homeassistant.components.sensor.updater
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Sensor that checks for available updates.
+
+For more details about this platform, please refer to the documentation at
+at https://home-assistant.io/components/sensor.updater/
+"""
+import logging
+
+import requests
+
+from homeassistant.const import __version__ as CURRENT_VERSION
+from homeassistant.helpers import event
+
+_LOGGER = logging.getLogger(__name__)
+PYPI_URL = 'https://pypi.python.org/pypi/homeassistant/json'
+DEPENDENCIES = []
+DOMAIN = 'updater'
+
+
+def setup(hass, config):
+    ''' setup the updater component '''
+
+    def check_newest_version(_=None):
+        ''' check if a new version is available and report if one is '''
+        newest = get_newest_version()
+        if newest != CURRENT_VERSION and newest is not None:
+            hass.states.set(
+                '{}.Update'.format(DOMAIN), newest,
```

Comment:
Entity ID is static, can you move it to a constant. Also, entity ids should be lowercase.